lib/routes.js

```diff
@@ -26,12 +26,23 @@ const argv = require('minimist')(process.argv.slice(2))
 
 const CLOCK_SKEW = 60 * 1000
 const REQUEST_TTL = 3 * 60 * 1000
+const THROTTLE_LOGS_QUERY = 5 * 60 * 1000
+const THROTTLE_CLOCK_SKEW = 1 * 60 * 1000
 
 const pids = {}
 const reboots = {}
 
 const devMode = argv.dev || options.http
 
+_.mixin({
+  memoizeThrottle: function(wait=0, func) {
+    var mem = _.memoize(function(param) {
+      return _.throttle(wait, func)
+    });
+    return function(param){mem(param)(param)}
+  }
+});
```

Comment:
```suggestion
})
```

Make sure the code passes standard.